api/src/main/java/org/pmiops/workbench/access/UserAccessModuleMapper.java

```diff
@@ -1,36 +1,20 @@
 package org.pmiops.workbench.access;
 
 import java.sql.Timestamp;
-import javax.annotation.Nullable;
-import org.mapstruct.AfterMapping;
-import org.mapstruct.Context;
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
-import org.mapstruct.MappingTarget;
-import org.mapstruct.NullValueMappingStrategy;
 import org.pmiops.workbench.db.model.DbUserAccessModule;
 import org.pmiops.workbench.model.AccessModuleStatus;
 import org.pmiops.workbench.utils.mappers.CommonMappers;
 import org.pmiops.workbench.utils.mappers.MapStructConfig;
 
 @Mapper(
     config = MapStructConfig.class,
-    uses = {CommonMappers.class, AccessModuleNameMapper.class},
-    nullValueMappingStrategy = NullValueMappingStrategy.RETURN_DEFAULT)
+    uses = {CommonMappers.class, AccessModuleNameMapper.class})
```

Comment:
unrelated simplification - I can take this out of the PR if you like

Reply:
I didn't see any default being used ... usually better not to use one IMO, if avoidable.